README.md

```diff
@@ -1 +1,254 @@
-# Budget-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget-App
+
+
+
+## Description
+
+> The Ruby on Rails capstone project ([remember what they are?](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/capstone_intro.md)) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+- I created a Ruby on Rails application that allows the user to:
+
+  - register and log in, so that the data is private to them.
+  - introduce new transactions associated with a category.
+  - see the money spent on each category.
+
+### Screenshots 📸
+
+    Categories
+![](./app/assets/images/img1.png) 
+
+
+    Category-Details 
+![](./app/assets/images/img5.png) 
+
+
+
+## Learning objectives
+
+- Use ruby gems as software packages system.
+- Install Ruby on Rails framework.
+- Understand Rails RESTful design and router.
+- Use controllers to handle requests and render empty views.
+- Use params from browser request in a safe way.
+- Use preprocessed html file with embedded Ruby code.
+- Use layouts and templates for shared content.
+- Use database migration files to maintain database schema.
+- Use validations for models.
+- Secure app from n+1 problems.
+- Understand what ORM is.
+- Write SQL queries with ActiveRecord.
+- Set up associations between models.
+- uild a webapp that requires the user to log in.
+- Use devise gem for authentication.
+- Limit access to webapp resources based on authorization rules.
+- Analyze in writing why you have made a coding choice using one structure over another.
+
+## Live Demo 🔗
+
```

Comment:
- [ ] @evans22j kindly take a look at your live demo, it seems your app deployment has some errors in Heroku, I am getting this error when I try to login or signup:
![image](https://user-images.githubusercontent.com/61408860/185747055-25397f43-a71d-401b-828a-c52dc6ec06c7.png)

`hint` You can log in to your Heroku account then your app page, under overview it provides you a chance to view your build log, this will help you debug the problem in your app.
<img width="932" alt="image" src="https://user-images.githubusercontent.com/61408860/185747201-a13a920c-a329-4660-9960-a563b6f29b44.png">